addons/spreadsheet/static/src/chart/plugins/odoo_chart_core_plugin.js

```diff
@@ -86,15 +86,7 @@ export class OdooChartCorePlugin extends CorePlugin {
      * @returns {Array<string>}
      */
     getOdooChartIds() {
-        const ids = [];
-        for (const sheetId of this.getters.getSheetIds()) {
-            ids.push(
-                ...this.getters
-                    .getChartIds(sheetId)
-                    .filter((id) => this.getters.getChartType(id).startsWith("odoo_"))
-            );
-        }
-        return ids;
+        return Object.keys(this.charts).filter((id) => this.getters.getChartType(id).startsWith("odoo_"));
```

Comment:
that's funny, because if you take `Object.keys(this.charts)`, it only contains odoo charts, so the `filter` should be useless

Reply:
Agreed! Pushed the changes and updated the commit as well.